config/examples/redhat_customer_portal.yml

```diff
@@ -1,19 +1,15 @@
 base: &default
-  enabled: true
-  client_id:
-  client_secret:
-  realm:
-  flow:
-  entitlements_notifications_enabled: true
+  enabled: <%= ENV.fetch('RH_CUSTOMER_PORTAL_ENABLED', false) %>
```

Comment:
I missed this last time,  `RH_CUSTOMER_PORTAL_ENABLED` seems to require converting to boolean one way or another